Improve code assist docs (#1965)
Qiskit/documentation: docs/guides/qiskit-code-assistant-jupyterlab.mdx

### Generate code

While you develop your code using Qiskit, you can ask to the Qiskit Code Assistant to help you. In general, the assistant suggests better code as response to Python comments or docstrings, but you can use the assistant anywhere in your file.

To get a code suggestion, type a prompt, then type `Alt` + `.` or `Alt` + `\`.  There are two types of prompts you can use:

  - Enter partial code and get a suggestion to finish the code.  Example:
    ```python
    from qiskit.circuit import QuantumCircuit
    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
    from qiskit_ibm_runtime import SamplerV2 as Sampler
    from qiskit_ibm_runtime.fake_provider import FakeManilaV2

    # Bell Circuit
    qc = QuantumCircuit(2)
    qc.h(0)
    qc.cx(0, 1)

    qc.mea # Type Ctrl + . for a code suggestion.
    ```
  - Type a pound (#) sign, then type your prompt.  Example: `# Give me a random circuit of 5 qubits and a depth of 4.`

### Work with code suggestions

Use the following to accept, reject, and cycle through suggestions:

> - `Alt` + `[` and `Alt` + `]` can be used to cycle through the list of suggestions if there are
>   more than one.

Reviewer: ```suggestion
- `Alt` + `[` and `Alt` + `]` can be used to cycle through the list of suggestions (if there are
  more than one).
```

## Troubleshooting

If you see the frontend extension but it is not working, check that the server
extension is enabled:

```bash
jupyter server extension list
```

If the server extension is installed and enabled, but you don't see the frontend

> extension, check the frontend extension is installed:

Reviewer: ```suggestion
extension, check that the frontend extension is installed:
```

## Contribute to the JupyterLab extension

The code for this extension is publicly available and open source. Check it out in [GitHub.](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab)

## Next steps

<Admonition type="tip" title="Recommendations">

> Check out examples to use the Qiskit Code Assistant for [circuits](https://docs.quantum.ibm.com/guides/circuit-library), [configuring error suppression](https://docs.quantum.ibm.com/guides/configure-error-suppression), and [transpiling with passmanagers.](transpile-with-pass-managers)

Reviewer: ```suggestion
See examples to use the Qiskit Code Assistant for [circuits]/guides/circuit-library), [configuring error suppression](/guides/configure-error-suppression), and [transpiling with pass managers.](/guides/transpile-with-pass-managers)
```
